polling_stations/apps/addressbase/helpers.py

```diff
@@ -18,6 +19,84 @@ def centre_from_points_qs(qs):
     return poly.centroid
 
 
+AddressTuple = namedtuple('Address', [
+    'address',
+    'postcode',
+    'council',
+    'polling_station_id',
+    'slug'])
+
+
+class AddressSet:
```

Comment:
Could this just subclass `set`, and only define the `save` method? That would allow us to use all the built in set methods without having to access `self.elements`